quoting-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Quoting-Service Helm chart for Kubernetes
 name: quoting-service
-version: 8.7.1
```

Comment:
this would need to be updated depending on the below comment and then this would need to be updated in the requirements.yaml under mojaloop folder